Tasks/InstallSSHKey/installsshkey-util.ts

```diff
@@ -14,6 +18,99 @@ export const postKnownHostsDeleteFileSetting: string = 'INSTALL_SSH_KEY_KNOWN_HO
 export const sshAgentPidEnvVariableKey: string = 'SSH_AGENT_PID';
 export const sshAgentSockEnvVariableKey: string = 'SSH_AUTH_SOCK';
 
+function execSshAddPassphraseSync(tool, args, passphrase):Q.Promise<boolean> {
```

Comment:
This looks like a copy of the exec() code without piping but setting stdin. I am ok with this but it might make sense adding support into the vsts-task-lib to be able to pass in an inputStream. Check with @bryanmacfarlane.